modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -80,10 +80,10 @@ Cordova::
 +
 [source,bash]
 ----
-$ npm install @aerogear/auth
+$ npm install @aerogear/auth --save
```

Comment:
@austincunningham  'save' hasn't been necessary for a few years https://blog.npmjs.org/post/161081169345/v500
do we need to specify it?

Reply:
My reading of it as long as not using shrinkwrap. it should be fine to removed `--save`, as that is default behavior. 
:+1:

Reply:
Thanks for the feedback @austincunningham, and good spot @finp .

Change made: https://github.com/aerogear/mobile-docs/pull/676/commits/7121c2f3738e17a279a66fc756b97c9f04cf5a83